libs/features/profile/src/lib/features-profile.tsx

```diff
@@ -113,28 +125,36 @@ export const FeaturesProfile = (props) => {
   const addCourse = (data: CourseData) => {
     setAddDrawer(false);
     dispatch(coursesActions.addCourse(data));
-  }
+  };
 
   return (
     <div className="features-profile">
       <div className="profile-card-container">
         <SharedUserProfileCard {...data} toggleDrawer={toggleDrawer} />
       </div>
       <div className="content">
-        <div className='course-button'>
-        <Button variant="contained" color="primary" onClick={() => setAddDrawer(true)}>
-          Add Course
-        </Button>
+        <div className="course-button">
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => setAddDrawer(true)}
+          >
+            Add Course
+          </Button>
         </div>
 
-        <Drawer anchor="right" open={addDrawer} onClose={() => setAddDrawer(false)}>
-            <SharedCreateCourseForm
-              onSubmit={addCourse}
-              onCancel={() => {}}
-              categories={categories}
-              lessons={lessons}
-              toggleAddDrawer={() => setAddDrawer(false)}
-            />
+        <Drawer
+          anchor="right"
+          open={addDrawer}
+          onClose={() => setAddDrawer(false)}
+        >
+          <SharedCreateCourseForm
+            onSubmit={addCourse}
+            onCancel={() => {}}
```

Comment:
this function do nothing, remove it and if needed in SharedCreateCourseForm do as optional parameter